src/plugins/visualizations/public/expressions/visualization_function.ts

```diff
@@ -94,6 +100,9 @@ export const visualization = (): ExpressionFunctionVisualization => ({
     const uiStateParams = args.uiState ? JSON.parse(args.uiState) : {};
     const uiState = new PersistedState(uiStateParams);
 
+    const aggConfigsState = args.aggConfigs ? JSON.parse(args.aggConfigs) : {};
+    const aggConfigs = indexPattern ? getSearch().aggs.createAggConfigs(indexPattern, aggConfigsState) : null;
```

Comment:
Rather than setting to `null` here, perhaps we should simply exclude the aggs from the `requestHandler` if for some reason no index pattern is present? Then they are basically an optional parameter?

Reply:
So I set `undefined` as default, instead of `null`?

Reply:
Yeah, I guess that's what I'm proposing. Basically so that the argument to the request handler is `aggs?: IAggConfigs` instead of `aggs: IAggConfigs | null`

I don't feel strongly about this though. In the end it won't make a huge difference since the request handler will likely check `if (aggs) {...}` the same either way.